.github/workflows/ci-sage.yml

```diff
@@ -98,3 +98,13 @@ jobs:
       # 'Package "sage-docker-..." is already associated with another repository.'
       docker_push_repository: ghcr.io/${{ github.repository }}/scip_sdp_
     needs: [dist]
+
+  macos:
+    # Use https://github.com/sagemath/sage/pull/37237
+    uses: mkoeppe/sage/.github/workflows/macos.yml@ci-macos-2024
+    with:
```

Comment:
@pfetsch This is ready for merging if you'd like to add macOS testing to your project.

I'll send a PR later to replace these temporary refs by permanent once.